packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js

```diff
@@ -647,13 +677,19 @@ class AccessibilityActionsExample extends React.Component {
   }
 }
 
-class FakeSliderExample extends React.Component {
-  state = {
+class FakeSliderExample extends React.Component<
+  {},
+  {
+    current: number,
+    textualValue: 'center' | 'right' | 'left',
+  },
+> {
+  state: {current: number, textualValue: 'center' | 'left' | 'right'} = {
```

Comment:
If you want, you could extract this state out into:
```
type FakeSliderExampleState = {
  current: number,
  textualValue: 'center' | 'left' | 'right',
}
```
so that you don't need to repeat this code twice!

Reply:
Gotcha! 💯